include/gz/msgs/actor.pb.h

```diff
@@ -16,3 +16,4 @@
  */
 
 #include <ignition/msgs/actor.pb.h>
+#include <ignition/msgs/config.hh>
```

Comment:
I'm not sure if this is going to work since it's on the `gz` headers for now

Reply:
The generated ignition messages use the ignition namespace, so that's here to ensure ignition::msgs works